docx/bookmark.py

```diff
@@ -58,7 +58,23 @@ def bookmark_pairs(self):
 class _PartBookmarkFinder(object):
     """Provides access to bookmark oxml elements in a story part."""
 
+    def __init__(self, part):
+        self._part = part
+
     @classmethod
     def iter_start_end_pairs(cls, part):
         """Generate each (bookmarkStart, bookmarkEnd) in *part*."""
+        return cls(part)._iter_start_end_pairs()
+
+    def _iter_start_end_pairs(self):
+        """Generate each (bookmarkStart, bookmarkEnd) in this part."""
+        for idx, bookmarkStart in self._iter_starts():
+            raise NotImplementedError
```

Comment:
The `NotImplementedError` should appear as the only line in this method. Otherwise you'd be testing more than one unit.